bin/run_atcs_mock.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# This file is part of ts_observatory_control.
+#
+# Developed for the LSST Telescope and Site.
+# This product includes software developed by the LSST Project
```

Comment:
Please refer to `Vera Rubin Observatory` instead of `LSST` in this and all other files.